Submitting solution for AlayaCare backend python test.

gilberdev opened the pull request.

flacours commented:
Hi,
Thank you for your good work!

A couple of comments:
- is the code from orm.py from you or you have to use something existing? if you use something existing it's ok but usually you can do a pip install of the library.
- authenticated user can modify todos of other users.
- i would have like to see the more recent todos first. right now they appear last in the list
- the paginator UI shows many pages if we have only 2 page

gilberdev commented:
Hi, @flacours. 
Thank you for your comments and sorry for taking so long in answering them. 

* The code in orm.py is mine.

* About authenticated users modifying other users todos: I already submitted a commit that fixes this problem.

* I implemented a QuerySet class that allows sorting his elements based on a group of columns. This option gives the possibility to sort the user todos in a descending way based on their id, which produces the effect of having the newer todos first.

*  This was just personal choice, however, I modified the implementation and left simple blog pagination of two buttons and a label that shows the actual pages and the total pages. I think that the current pagination will be closer to what you expected.
